web/src/components/ui/sidebar.tsx

```diff
@@ -58,7 +58,7 @@ const SidebarProvider = React.forwardRef<
 >(
   (
     {
-      defaultOpen = true,
+      defaultOpen = false,
```

Comment:
@marcklingen I would suggest collapsing the sidebar by default, rather than showing the expanded state. My suggestion is based on the assumption that users typically prefer the wider screen size when viewing items and and can easily access the navigation in the collapsed state as well. Navigation is still intuitive and on-click (max 2 click for nested items) so I'd feel it's net positive to collapse. Let me know what you think :)